tools/python_api/CMakeLists.txt

```diff
@@ -3,17 +3,21 @@ project(_kuzu)
 
 set(CMAKE_CXX_STANDARD 20)
 
+file(GLOB SOURCE_PY
```

Comment:
I would rather not use GLOB, but list all necessary files here.

Reply:
I have changed the C++ target to use the explicit path instead of using GLOB. But I think we can keep the GLOB for Python files since CMake is not actually running any build script against the Python files. It is just copying the files to the build directory.